eng/pipelines/runtime-richnav.yml

```diff
@@ -0,0 +1,43 @@
+trigger:
+  branches:
+    include:
+    - master
+    - dev/infrastructure
+    - release/*.*
+
+pr: none
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+- name: TeamName
+  value: dotnet-core-acquisition
+
+stages:
+- stage: Build
+  jobs:
+  #
+  # Checkout repository
+  #
+  - template: /eng/pipelines/common/checkout-job.yml
+
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/common/global-build-job.yml
+      buildConfig: Checked
+      platforms:
+      - Windows_NT_x64
+      jobParameters:
+        enableRichCodeNavigation: true
+        timeoutInMinutes: 120
+        buildArgs: -s libs.ref+libs.src -c debug -runtimeConfiguration Checked
```

Comment:
Let's remove `-runtimeConfiguration` from both of these guys... debug for all is fine.

Reply:
Also, are we going to pass down the MSBuild arg to restore the package?

`/p:EnableRichCodeNavigation=true`?

We could do that directly in `global-build-job.yml` as we do for other things.
https://github.com/dotnet/runtime/blob/7601d44ca06fc977c4a3c7a33c1fc864f8f5ad78/eng/pipelines/common/global-build-job.yml#L53-L57

we could add something like:
```yml
      - name: _richCodeNavigationParam
        ${{ if eq(parameters.enableRichCodeNavigation, true) }}:
          value: /p:EnableRichCodeNavigation=true
        ${{ if ne(parameters.enableRichCodeNavigation, true) }}:
          value: ''
```

and then flow it here: https://github.com/dotnet/runtime/blob/7601d44ca06fc977c4a3c7a33c1fc864f8f5ad78/eng/pipelines/common/global-build-job.yml#L78

Reply:
Ah yes. This is done in the [template](https://github.com/dotnet/runtime/blob/master/eng/common/templates/job/job.yml#L68..L70), but I agree with you, for repro-ability/readability I think we should pipe it through as well. Thanks for the detailed description, I'll make this change!